typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -38,44 +75,71 @@ export class EvmIsmModule extends HyperlaneModule<
   ) {
     super(args);
     this.reader = new EvmIsmReader(multiProvider, args.chain);
-    this.creator = new EvmIsmCreator(deployer, multiProvider, args.addresses);
   }
 
   public async read(): Promise<IsmConfig> {
     return await this.reader.deriveIsmConfig(this.args.addresses.deployedIsm);
   }
 
   public async update(config: IsmConfig): Promise<EthersV5Transaction[]> {
-    throw new Error('Method not implemented.');
-
     const destination = this.multiProvider.getChainName(this.args.chain);
-    await this.creator.update({
-      destination,
-      config,
-      existingIsmAddress: this.args.addresses.deployedIsm,
-    });
+
+    if (typeof config === 'string') {
+      this.logger.debug('Skipping update for config of type Address.');
+      return [];
+    }
```

Comment:
spent 15mins looking into this and it may conflict with other modules, so I'll wait til we have ism+core+hook+warp merged before PR'ing this

Created issue to track https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3773